src/main/java/com/fouristhenumber/utilitiesinexcess/common/blocks/BlockEtherealGlass.java

```diff
@@ -0,0 +1,101 @@
+package com.fouristhenumber.utilitiesinexcess.common.blocks;
+
+import java.util.List;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockGlass;
+import net.minecraft.block.material.Material;
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.AxisAlignedBB;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public class BlockEtherealGlass extends BlockGlass {
+
+    public BlockEtherealGlass() {
+        super(Material.glass, false);
+        setBlockName("etherealGlass");
+    }
+
+    @SideOnly(Side.CLIENT)
+    private IIcon[] icons;
+
+    @Override
+    public void addCollisionBoxesToList(World worldIn, int x, int y, int z, AxisAlignedBB mask,
+        List<AxisAlignedBB> list, Entity collider) {
+        int meta = worldIn.getBlockMetadata(x, y, z);
+
+        if ((meta == 0 || meta == 2) && collider instanceof EntityPlayer) {
+            return;
+        }
+
+        if ((meta == 3 || meta == 5) && !(collider instanceof EntityPlayer)) {
+            return;
+        }
+
+        super.addCollisionBoxesToList(worldIn, x, y, z, mask, list, collider);
+    }
+
+    @Override
+    public int getLightOpacity(IBlockAccess world, int x, int y, int z) {
+        int meta = world.getBlockMetadata(x, y, z);
+        return (meta == 0 || meta == 3) ? 0 : 255;
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public void getSubBlocks(Item itemIn, CreativeTabs tab, List<ItemStack> list) {
+        list.add(new ItemStack(this, 1, 0)); // Normal
```

Comment:
Can you define the metas in an enum/constants instead of hardcoding magic numbers?